feedback/20220614.md

```diff
@@ -0,0 +1,35 @@
+# First Signals from Dev Community
+
+> see [communication-management/issues/111](https://github.com/blindnet-io/communication-management/issues/111)
+
+## Direct Feedback
+
+### Big Corps Still Need Guidance
+
+Even when a big company _has_ a security/privacy expertise team, some other teams might not have access to it, and can still therefore benefits from using our tools.
+
+### Identity management
+
+Identity and privacy are strongly linked. Especially in a context where cross-referring data between a wide variety of context allow inferring very personal information.
+
+The "v1" approach (i.e. identifying a submitter only by a unique id linked to one data collection) _does_ make a lot of sense in this context.
+
+However, there are still two issues with this approach:
+
+1. Privateform relies heavily on email address for RGPD rights management, which can't be considered a secured solution, as it can be easily stolen
+2. no real solution is provided to help submitters to easily manage the private information they submitted
+
+Users (and especially developers) focus a lot on solution easing identity management, as ease of use stay, for now, more important than actual trust.
+And using emails reduce trust.
+
+### Certifications / Convincing Managers
+
+Most middle-sized and big companies still relies on certifications to determine which infosec solution to choose.
```

Comment:
It would be usefull to know here:

1. what are the exact certifications that are expected, and
2. would that apply to blindnet devkit as focused much more on Data Subject Rights Management (than on infosec)